### ISSUE TITLE
Add a confirm step when erasing a user

### ISSUE THREAD

**Issue by XaviP** [FreedomCoop/valuenetwork#105]
This is a request from admin Maro

**Comment by XaviP** [comment]
Related to https://github.com/FreedomCoop/valuenetwork/issues/69

**Comment by bhaugen** [comment]
The admin team is right!

**Comment by mario567** [comment]
would be nice to have a confirm step/window for any deletion button in ocp, not just for erasing user

**Comment by bhaugen** [comment]
I agree with all of the comments above. In the meantime, you need to be a superuser to delete a user. An admin can delete an agent, but I just made that more difficult, too. OCP has a bunch of checks that must be passed for agent to be deletable. If they are not deletable, you will not see the delete button at all.
Those checks involved whether the agent had done anything in OCP, because if so, deleting them would orphan that history. I just added a check to see if they had a faircoin address, which means all agents created with a user after the faircoin addresses were automatically created. 

So after that change is deployed, you will not be able to delete any of those agents anymore. Let me know if you disagree with that.

We'll still add delete confirmations all over the place, but that should prevent some accidental deletions for now.

**Comment by bum2** [comment]
Is ok like you say about agents. We can always change the relation to any other agent putting status 'inactive' or changing the type of relation or deleting the relation.
Let's check what other things can be deleted in work app or coopadmin app and are missing a confirm step.
